pkgs/by-name/sc/scrounge-ntfs/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, stdenv
+, fetchurl
+, autoreconfHook
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "scrounge-ntfs";
+  version = "0.9";
+
+  src = fetchurl {
+    url = "http://thewalter.net/stef/software/scrounge/scrounge-ntfs-${finalAttrs.version}.tar.gz";
+    hash = "sha256-HYrMIMTRPmgAac/vaZ1jaUFchyAl5B0quxgHH0DHJ84=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
```

Comment:
there is already a configure script so the build will work without this. is there a reason you're adding this hook?